### PR TITLE
Update branding to 3.1.1

### PR DESCRIPTION
**Pull request by wtgodbe**
Update branding & add trigger for internal release/3 branches

@JunTaoLuo @mmitche PTAL

### CODE REVIEW

File: .azure/pipelines/ci.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ trigger:
     include:
     - master
     - release/*
+    - internal/release/3.*
```
Comment (JunTaoLuo) [inline .azure/pipelines/ci.yml]: I thought we were only supposed to update the PR section and not the trigger section

Comment (wtgodbe) [reply .azure/pipelines/ci.yml]: Originally that was the case, but @mmitche and I have since come up with a new scheme & changed our minds on this one.

**Review by JunTaoLuo — APPROVED** [review]

**Comment by wtgodbe** [comment]
This will probably fail until we get https://github.com/aspnet/AspNetCore/pull/17311 in & merge it to 3.1

**Comment by wtgodbe** [comment]
Template failures:

> Project new angular --auth Individual failed to publish.\r\nProcess exited with code 1\nStdErr: \nStdOut: Microsoft (R) Build Engine version 16.4.0-preview-19475-01+497296502 for .NET Core\r\nCopyright (C) Microsoft Corporation. All rights reserved.\r\n\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-preview1-014400\MSBuild.dll -maxcpucount -property:Configuration=Release -restore -target:Publish -verbosity:m /bl .\AspNet.angularindividual.oueksvxrynk.csproj\r\n Restore completed in 23.26 ms for F:\workspace.16\_work\1\s\src\ProjectTemplates\test\bin\Release\netcoreapp3.1\TestTemplates\AspNet.angularindividual.oueksvxrynk\AspNet.angularindividual.oueksvxrynk.csproj.\r\n You are using a preview version of .NET Core. See: https://aka.ms/dotnet-core-preview\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-preview1-014400\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3106: Assembly strong name "F:\workspace.16\_work\1\s\obj\template-restore\microsoft.aspnetcore.app.ref\3.0.0\ref\netcoreapp3.1\Microsoft.AspNetCore.Antiforgery.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\). [F:\workspace.16\_work\1\s\src\ProjectTemplates\test\bin\Release\netcoreapp3.1\TestTemplates\AspNet.angularindividual.oueksvxrynk\AspNet.angularindividual.oueksvxrynk.csproj]\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-preview1-014400\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3106: Assembly strong name "F:\workspace.16\_work\1\s\obj\template-restore\microsoft.aspnetcore.app.ref\3.0.0\ref\netcoreapp3.1\Microsoft.AspNetCore.Authentication.Abstractions.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\). [F:\workspace.16\_work\1\s\src\ProjectTemplates\test\bin\Release\netcoreapp3.1\TestTemplates\AspNet.angularindividual.oueksvxrynk\AspNet.angularindividual.oueksvxrynk.csproj]\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-preview1-014400\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3106: Assembly strong name "F:\workspace.16\_work\1\s\obj\template-restore\microsoft.aspnetcore.app.ref\3.0.0\ref\netcoreapp3.1\Microsoft.AspNetCore.Authentication.Cookies.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\). [F:\workspace.16\_work\1\s\src\ProjectTemplates\test\bin\Release\netcoreapp3.1\TestTemplates\AspNet.angularindividual.oueksvxrynk\AspNet.angularindividual.oueksvxrynk.csproj]\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-preview1-014400\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3106: Assembly strong name "F:\workspace.16\_work\1\s\obj\template-restore\microsoft.aspnetcore.app.ref\3.0.0\ref\netcoreapp3.1\Microsoft.AspNetCore.Authentication.Core.dll" is either a path which could not be found or it is a full assembly name which is badly formed. If it is a full assembly name it may contain characters that need to be escaped with backslash(\). Those characters are Equals(=), Comma(,), Quote("), Apostrophe('), Backslash(\). [F:\workspace.16\_work\1\s\src\ProjectTemplates\test\bin\Release\netcoreapp3.1\TestTemplates\AspNet.angularindividual.oueksvxrynk\AspNet.angularindividual.oueksvxrynk.csproj]\r\nF:\workspace.16\_work\1\s\.dotnet\sdk\3.1.100-previe

**Comment by wtgodbe** [comment]
Others are `PlatformManifestListsAllFiles` and `AssembliesAreReferenceAssemblies`, which I think we fixed in 3.0. Looking

**Comment by wtgodbe** [comment]
Oh, we need to update the baseline.

**Comment by wtgodbe** [comment]
This will fail at least until `dotnet-install.sh` gets fixed

**Pull request merged**